data/tlg0011/tlg003/__cts__.xml

```diff
@@ -1,9 +1,15 @@
 <?xml version="1.0" encoding="UTF-8"?>
 <ti:work xmlns:ti="http://chs.harvard.edu/xmlns/cts" urn="urn:cts:greekLit:tlg0011.tlg003" xml:lang="grc" groupUrn="urn:cts:greekLit:tlg0011">
    <ti:title xml:lang="eng">Ajax</ti:title>
+   
    <ti:edition xml:lang="grc" workUrn="urn:cts:greekLit:tlg0011.tlg003" urn="urn:cts:greekLit:tlg0011.tlg003.1st1K-grc1">
-      <ti:label xml:lang="lat">Sophoclis Aiax. Commentario perpetuo illustravit Chrst. Augustus Lobeck</ti:label>
-      <ti:description xml:lang="eng">Sophocles, creator; Lobeck Christian, editor</ti:description>
+      <ti:label xml:lang="grc">Αἴας μαστιγοφόρος</ti:label>
+      <ti:description xml:lang="mul">Sophocles. Sophoclis Aiax. Lobeck Christian, editor. Berlin: Weidmann, 1866.</ti:description>
+   </ti:edition>
+   
+ <ti:edition xml:lang="grc" workUrn="urn:cts:greekLit:tlg0011.tlg003" urn="urn:cts:greekLit:tlg0011.tlg003.opp-grc1">
```

Comment:
Any chance @lcerrato that we could make this URN tlg0011.tlg003.1st1K-grc2. I've been retiring opp-grc URNs found within Scaife/OGL and having opp-grc URNs refer exclusive to editions linked to by the Perseus Catalog if that makes sense.

Reply:
Sure, I was not certain about how/when this was assigned. I'm just trying to clean up old issues for the new year.